grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -283,6 +286,101 @@ class StockMovementController {
         render(template: "documents", model: [stockMovement: stockMovement])
     }
 
+    def comments = {
+        def stockMovement = getStockMovement(params.id)
```

Comment:
what's the difference in using that and using the one from the `stockMovementService`? Is there any case that you use this one instead of the one from the service?

Reply:
There was one previous comment which addressed the difference and I had to change them to use `stockMovementService.getStockMovement`, must have missed this one because the rest are using `stockMovementService.getStockMovement`

Reply:
@kchelstowski @drodzewicz Some time ago I added this utility method in this controller (while I was fixing packing list, documents and receipts tab for returns), but I do not remember why exactly
 ```groovy
// Used by SM show page 'tabs' actions - packing list, documents and receipts
def getStockMovement(String stockMovementId) {
    ...
}
```

Reply:
Looks like it was some sort of performance improvement (see https://pihemr.atlassian.net/browse/OBPIH-4544)